app/src/lib/app-state.ts

```diff
@@ -316,11 +316,15 @@ export type Foldout =
   | { type: FoldoutType.AddMenu }
   | AppMenuFoldout
 
-export enum RepositorySection {
+export enum RepositorySectionTab {
   Changes,
   History,
 }
 
+export type RepositorySection =
+  | { selectedTab: RepositorySectionTab.Changes }
+  | { selectedTab: RepositorySectionTab.History; focusBranchSelector: boolean }
```

Comment:
Three things:

:art: can this name here also mention the `initial` prefix?
:art: can `focusBranchSelector` here be optional, and default to `false` when doing the lookup?
:art: can you split this type signature over two lines to avoid the `;`, or is this Prettier making me sad?

Reply:
I can't assign a default value in a type alias definition. I had to handle the `undefined` case in the component that uses that property.